RadixWallet/Clients/TokenPriceClient/TokenPriceClient+Live.swift

```diff
@@ -0,0 +1,74 @@
+extension TokenPricesClient {
+	public static let liveValue: TokenPricesClient = {
+		@Dependency(\.httpClient) var httpClient
+		@Dependency(\.jsonDecoder) var jsonDecoder
+		@Dependency(\.jsonEncoder) var jsonEncoder
+		@Dependency(\.cacheClient) var cacheClient
+		let rootURL = URL(string: "https://dev-token-price.extratools.works")!
```

Comment:
are we going to do an `#if DEBUG` for this base URL?

Reply:
shoot, forgot that I need to use the prod server in release mode. Thank you for catching this!